app/controllers/HasSupplementaryUnitController.scala

```diff
@@ -49,8 +49,6 @@ class HasSupplementaryUnitController @Inject() (
 
   def onPageLoad(mode: Mode, recordId: String): Action[AnyContent] = (identify andThen getData andThen requireData) {
     implicit request =>
-      //TODO this definitely will not survive in the new world
```

Comment:
again have we done a fix for this?

Reply:
There is a seperate refactoring ticket for this